vaadin-dev-server/src/main/java/com/vaadin/base/devserver/OpenInCurrentIde.java

```diff
@@ -208,7 +215,8 @@ private static String getIdeaBinary(Info info) {
                 }
             }
         }
-        if (info.command().get().contains("idea")) {
+        if (info.command().isPresent()
+                && info.command().get().contains("idea")) {
             return info.command().get();
         }
         return null;
```

Comment:
nit: I would use a filter here, but current solution is also fine
```suggestion
        return info.command()
                .filter(cmd -> cmd.contains("idea"))
                .orElse(null);
```

Reply:
Thanks, done